src/main/java/com/couchbase/lite/replicator/ReplicationInternal.java

```diff
@@ -1314,11 +1319,6 @@ protected void retryReplicationIfError(){
         }
     }
 
-    // TODO: not used, please remove
-    protected void updateActive() {
-        Log.v(Log.TAG_SYNC, "%s: updateActive() called", this);
-    }
-
```

Comment:
As the updateAction() method is now removed, we should update some log messages in the file that say something about calling updateActive() method to avoid confusion.